index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Welcome to my portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="AppBar app">
+        <div class="logo">
+            <p id="name">Rugwizangoga</p>
+        </div>
+        <div class="menu">
+            <img src="/pic/Icons/Enabled.png" alt="menu">
+        </div>
+    </header>
+    <main class="app">
+        <div class="headline">
+            <div class="primarytext app">
+                <p>Hey there.
+                    I'm Pacific</p>
+            </div>
+            <div class="supportingtext app">
+                <p>I’am a Software Developer</p>
+            </div>
+            <div class="description app">
+                <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.</p>
+            </div>
+            <div class="socialmedia app">
+                <div class="icon">
+                    <a href="https://github.com/rugwizangoga"><img src="/pic/Icons/hub.png" alt="GitHub"></a>
+                </div>
+                <div class="icon">
+                    <a href="https://www.linkedin.com/in/iradukunda-pacific-rugwizangoga"><img src="/pic/Icons/in.png" alt="LinkeIn"></a>
+                </div>
```

Comment:
- [ ] [REQUIRED] I think a more semantic way of listing your communication social media. You can use `ul` and `li` instead of `div`s.

## example

```html
<ul>
  <li>
    <a href="#">
      <img src="./path" />
    </a>
  </li>
  <!-- rest of code -->
</ul>
```